mantidproject/mantid
# Added 2nd order Placzek correction to POLARIS Total Scattering

**Description of work.**

Add support for the 2nd order term in the Placzek correction to the ISIS total scattering workflow. The second order term was added by the ORNL team in the algorithm CalculatePlaczek so this task plumbs that into the ISIS workflow.

As discussed with Helen Playford

**To test:**

Run this script and check it completes successfully. It is a varation of the script on https://github.com/mantidproject/mantid/pull/34193. There is a zip file attached to that PR containing some config files that you'll need for this to work. The `config_file_path` in the script below needs updating to match the location where you unzip the files:

```
# import mantid algorithms, numpy and matplotlib
from mantid.simpleapi import *
import matplotlib.pyplot as plt
import numpy as np

from isis_powder import polaris, SampleDetails

config_file_path = r"C:\Total Scattering\polaris_config_example.yaml"
polaris = polaris.Polaris(config_file=config_file_path, user_name="test", mode="PDF")

sample_details = SampleDetails(height=4.0, radius=0.2985, center=[0, 0, 0], shape='cylinder')
sample_details.set_material(chemical_formula='Si', packing_fraction=0.6)
polaris.set_sample_details(sample=sample_details)

polaris.create_vanadium(first_cycle_run_no="98532", multiple_scattering=False)

polaris.focus(run_number="98533", input_mode='Summed')

q_lim = [[2.5, 3, 4, 6, 7], [3.5, 5, 7, 11, 40]]

# default params for system test
polaris.create_total_scattering_pdf(run_number="98533", merge_banks=False,
                                    pdf_type="G(r)", lorch_filter=True, debug=True, 
                                    placzek_order=2, sample_temp=273)
```

Fixes #34196.


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
